src/main/java/org/flowsoft/flowg/WrongPointIndexingException.java

```diff
@@ -0,0 +1,28 @@
+package org.flowsoft.flowg;
+
+import java_cup.runtime.ComplexSymbolFactory.Location;
+
+public class WrongPointIndexingException extends Exception {
+        private final String _indexer;
+        private final Location _left;
+        private final Location _right;
+
+    public WrongPointIndexingException(String identifier, Location left, Location right) {
+            _indexer = identifier;
+            _left = left;
+            _right = right;
+    }
+
+        public String GetIdentifier() {
+            return _indexer;
+        }
+
+        public Location GetLeft() {
+            return _left;
+        }
+
+        public Location GetRight() {
+            return _right;
+        }
+}
+
```

Comment:
In the type checker we use the symbol not found exception for when you try to access a point entry that is not x, y, or z. Either use that or make the type checker use this